actors/builtin/market/market_actor.go

```diff
@@ -489,66 +489,77 @@ func (a Actor) CronTick(rt Runtime, params *adt.EmptyValue) *adt.EmptyValue {
 	rt.State().Transaction(&st, func() interface{} {
 		dbe, err := AsSetMultimap(adt.AsStore(rt), st.DealOpsByEpoch)
 		if err != nil {
```

Comment:
You don't need the `if err != nil` for `RequireNoErr`, it's handled internally.

Reply:
@anorth Will fix this next PR.